File: javascript/7kyu/Number of People in the Bus.js
// https://www.codewars.com/kata/5648b12ce68d9daa6b000099

var number = function(busStops){
    let peopleInBus = 0;
    busStops.forEach(([enter, left]) => {
        peopleInBus += enter - left;
    })
    return peopleInBus
}

File: java/7kyu/Number of People in the Bus.java
// https://www.codewars.com/kata/5648b12ce68d9daa6b000099

import java.util.List;

class Metro {
  int countPassengers(List<int[]> stops) {
    return stops.stream().mapToInt(x -> x[0] - x[1]).sum();
  }
}